content/slice/language-guide/result-types.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Result types
+description: Learn how to define and use results in Slice.
+---
+
+{% slice1 %}
+The built-in generic type Result<Success, Failure> is not available with Slice1.
+{% /slice1 %}
+
+{% slice2 %}
+
+## Result
+
+A result is a built-in generic type that holds either a Success value or a Failure value, where `Success` and `Failure`
+are type parameters.
+
+You construct a result inline, without giving it a name, usually to specify the return type of an operation. For
```

Comment:
This is the phrase we use on the sequence and dictionary pages, and I think it sounds better as well.
```suggestion
You can construct a result inline, without giving it a name, usually to specify the return type of an operation. For
```

Reply:
Fixes as suggested.